src/public/modules/forms/config/forms.client.routes.js

```diff
@@ -21,8 +21,9 @@ angular.module('forms').config([
           FormData: [
             'FormApi',
             '$transition$',
-            function (FormApi, $transition$) {
-              return FormApi.getPublic($transition$.params()).$promise
+            '$q',
+            function (FormApi, $transition$, $q) {
+              return $q.when(FormApi.getPublic($transition$.params().formId))
```

Comment:
I don't think we need `$q.when` in this file. you can see that the original code returned Promises via `.$promise` - this is because the routes are designed to load when the Promise resolves. can see the `state` documentation [here](https://ui-router.github.io/ng1/docs/1.0.0-beta.1/classes/state.stateprovider.html)